consul/enforce-min-tls-version/kyverno-test.yaml

```diff
@@ -0,0 +1,18 @@
+name: enforce-min-tls-version
+policies:
+  - enforce-min-tls-version.yaml
+resources:
+  - resource.yaml
+results:
+##### Pods - Bad
+  - policy: enforce-min-tls-version
+    rule: adding-capabilities
```

Comment:
rule name reference is incorrect

Reply:
fixed: rule updated to `check-for-tls-version`